VideoWeb/VideoWeb/Controllers/VideoEventsController.cs

```diff
@@ -82,29 +83,43 @@ public async Task<IActionResult> SendHearingEventAsync(ConferenceEventRequest re
             }
         }
 
-        private async Task SendEventToVideoApi(ConferenceEventRequest request)
+        private Task SendEventToVideoApi(ConferenceEventRequest request)
         {
-            if (request.EventType != EventType.VhoCall)
+            if (request.EventType == EventType.VhoCall)
             {
-                _logger.LogTrace("Raising video event: ConferenceId: {ConferenceId}, EventType: {EventType}",
-                        request.ConferenceId, request.EventType);
-
-                await _videoApiClient.RaiseVideoEventAsync(request);
+                return Task.CompletedTask;
             }
+
+            _logger.LogTrace("Raising video event: ConferenceId: {ConferenceId}, EventType: {EventType}",
+                    request.ConferenceId, request.EventType);
+
+            return _videoApiClient.RaiseVideoEventAsync(request);
         }
 
-        private async Task PublishEventToUi(ConferenceEventRequest request, Conference conference)
+        private CallbackEvent TransformAndPublishEventToUi(ConferenceEventRequest request, Conference conference)
```

Comment:
Doesn't look like you publish here anymore, rather map to a callback event only

Reply:
oops thanks :)